src/services/api/userProfile/getAllIncomeCategories.ts

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+import { expensesCategoriesAllEndPoint } from "../auth/apiConstants";
+import { ICategoriesTypes } from "../../../types/pages/Expenses";
+
+export const getAllIncomeCategories = async (baseUrl: string, data: ICategoriesTypes) => {
+	return await axios(`${baseUrl}/${expensesCategoriesAllEndPoint}`, {
+		method: "GET",
+		data: data,
```

Comment:
params: {
			is_income: data.is_income,
			is_outcome: data.is_outcome,
		},

can you replace data field with params field?